HW_4/task_2.py

```diff
@@ -0,0 +1,18 @@
+"""
+Задайте список случайных чисел. Выведите список чисел, которые не повторяются в заданном списке.
+
+Ввод: значение типа <list> (либо значения типа <int> – размерность списка)
+Вывод: значение типа <list>
+
+Пример:
+[1, 1, 2, 3, 3, 4, 5, 5, 6, 7, 7, 8, 9, 9]
+[2, 4, 6, 8]
+"""
+
+lst = [1, 1, 2, 3, 3, 4, 5, 5, 6, 7, 7, 8, 9, 9]
+#lst = list(map(int, input("Enter numbers separated by spaces:\n").split()))
+
+new_lst = [i for i in lst if lst.count(i) == 1]
```

Comment:
Чтобы немного уменьшить сложность, можно сделать так.
```suggestion
new_lst = [i for i in set(lst) if lst.count(i) == 1]
```
Само собой, если в списке нет повторяющихся элементов, то сложность это сильно не уменьшит.